modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/search_columns.jspf

```diff
@@ -14,6 +14,10 @@
  */
 --%>
 
+<%
+boolean hasUpdatePermission = PasswordPolicyPermissionUtil.contains(permissionChecker, passwordPolicy.getPasswordPolicyId(), ActionKeys.UPDATE);
+%>
```

Comment:
If it's not too difficult, let's try setting a single rowURL variable up here to either an actual value or null.  It can be a little tricky to do this when the URL is created with a taglib, but let's see if it's possible. That way there's only one place to actually change and maintain the logic.